ajax/libs/jquery-nearest/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "jquery-nearest",
+    "description": "Find the elements in a page that are closest to a particular point or element, based on pixel dimensions",
+    "filename": "jquery.nearest.min.js",
+    "keywords": [
+        "distance",
+        "ecosystem:jquery", 
+        "furthest", 
+        "jquery-plugin",         
+        "nearest",
+        "touching", 
+        "pixel"        
+    ],
+    "author": {
+        "name": "Gilmore Davidson",
+        "url": "https://shoehornwithteeth.com/"
+     },
+    "homepage": "https://gilmoreorless.github.io/jquery-nearest/",
+    "repository": {
+        "type": "git",      
+        "url": "git+https://github.com/gilmoreorless/jquery-nearest.git"
+    },    
+    "autoupdate": {
+        "source": "git",
+        "target": "git://github.com/gilmoreorless/jquery-nearest.git",
+        "fileMap": [
+          {
+            "basePath": "src",
+            "files": [
+              "jquery.nearest*.+(css|map|js)",
+              "jquery.nearest*"
```

Comment:
`jquery.nearest.js` in old versions is not under `src`. So we need to use add the pattern like [#discussion_r135428810](#discussion_r135428810) said. Thank you.

Reply:
@extend1994 
You have completely confused me. The files array now is looks like you asked to change... What I should do now?
@sufuf3 @PeterDaveHello

Reply:
@sashberd So sorry for confusing you. The full part of `autoupdate` field would be like
```javascript
"autoupdate": {
        "source": "git",
        "target": "git://github.com/gilmoreorless/jquery-nearest.git",
        "fileMap": [
          {
            "basePath": "src",
            "files": [
              "jquery.nearest*.+(css|map|js)"
            ]
          },
          {
            "basePath": "",
            "files": [
              "jquery.nearest*"
            ]
          }
        ]
    },
```